### PR TITLE
docs: Remove Zun documentation with kata containers

### LINKED ISSUE #3581: remove zun as a use case documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR removes the zun documentation use case with kata containers mainly
because is not longer valid as it is using as a reference docker with
clear containers 2.0 which are not longer being supported and it is also
using docker to test kata with openstack zun and docker is also not supported.

Fixes #3581

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by dborquez — APPROVED** [review]
LGTM thanks @GabyCT

File: docs/use-cases/zun_kata.md [hunk 1]
```diff
@@ -1,121 +0,0 @@
...
-
-Figure 3: Two BusyBox containers successfully launched
-
-![Kata Zun image 4](./images/kata-zun4.png)
-
-Figure 4: Test connectivity between Kata Containers
-
-![Kata Zun image 5](./images/kata-zun5.png)
-
-Figure 5: CLI for Zun
```
Comment (fidencio) [inline docs/use-cases/zun_kata.md:121]: @GabyCT, please, also remove those image files which are under https://github.com/kata-containers/kata-containers/tree/main/docs/use-cases/images

Comment (GabyCT) [reply docs/use-cases/zun_kata.md:121]: @fidencio changes applied

Comment (fidencio) [reply docs/use-cases/zun_kata.md:121]: Thanks!

**Review by fidencio — APPROVED** [review]
Personally, I'd have squashed those 2 commits into the same one, but it's fine as it is.
Thanks, @GabyCT!

**Pull request merged**